mozetl/clientsdaily/fields.py

```diff
@@ -97,9 +97,15 @@ def mode(l, empty_value='MISSING'):
         'plugins_notification_shown_sum'),
     # plugins_notification_user_action
     # popup_notification_stats
-    F.first('profile_creation_date').alias('profile_creation_date'),
-    F.sum('push_api_notification_received').alias(
-        'push_api_notification_received_sum'),
+    F.first(
+        F.expr(
+            "datediff(subsession_start_date, " \
```

Comment:
Nit: If you call the function directly instead of through `expr`, you can use a line break instead of using a line continuation.